<!-- -->
<launch>
  <arg name="lidar_points_topic" default="points_no_ground" />
  <arg name="radar_points_topic" default="radar/target_list_cartesian" />
  <arg name="vehicle_speed_topic" default="current_velocity" />
  <arg name="velocity_offset" default="1.2" />
  <arg name="decelerate_vel_min" default="1.3" />
  <arg name="remove_points_upto" default="2.3" />
  <arg name="enablePlannerDynamicSwitch" default="false" />

  <arg name="radar_detection_height_top" default="2" />
  <arg name="radar_detection_height_bottom" default="-0.3" />
  <arg name="deceleration_obstacle" default="0.7" />
  <arg name="deceleration_stopline" default="0.3" />
  <arg name="accelerate_max" default="1.3" />
  <arg name="velocity_change_limit" default="7.0" />
  <arg name="temporal_waypoints_size" default="100.0" />

  <node pkg="astar_planner" type="velocity_set" name="velocity_set" output="screen">
    <param name="lidar_points_topic" value="$(arg lidar_points_topic)" />
    <param name="radar_points_topic" value="$(arg radar_points_topic)" />
    <param name="vehicle_speed_topic" value="$(arg vehicle_speed_topic)" />
    <param name="velocity_offset" value="$(arg velocity_offset)" />
    <param name="decelerate_vel_min" value="$(arg decelerate_vel_min)" />
    <param name="remove_points_upto" value="$(arg remove_points_upto)" />
    <param name="enablePlannerDynamicSwitch" value="$(arg enablePlannerDynamicSwitch)" />

    <param name="radar_detection_height_top" value="$(arg radar_detection_height_top)" />
    <param name="radar_detection_height_bottom" value="$(arg radar_detection_height_bottom)" />
    <param name="deceleration_obstacle" value="$(arg deceleration_obstacle)" />
    <param name="deceleration_stopline" value="$(arg deceleration_stopline)" />
    <param name="accelerate_max" value="$(arg accelerate_max)" />
    <param name="velocity_change_limit" value="$(arg velocity_change_limit)" />
    <param name="temporal_waypoints_size" value="$(arg temporal_waypoints_size)" />
  </node>

</launch>
